tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -257,4 +257,52 @@ public function testOnlyOwnCommentIsParsed(): void
             ->getColumn('col1')
             ->getComment());
     }
+
+    public function testUnnamedForeignKeyConstraintHandling(): void
+    {
+        $sql = <<<SQL
+CREATE TABLE "bug4243_types" (
+"id" integer not null primary key autoincrement
+);
+CREATE TABLE "bug4243_groups" (
+"id" integer not null primary key autoincrement
+);
+CREATE TABLE "bug4243_documents" (
+    "hash" varchar not null,
+    "id" integer not null primary key autoincrement,
+    "created_at" datetime null,
+    "updated_at" datetime null,
+    "title" varchar not null,
+    "ek" varchar not null,
+    "iv" varchar not null,
+    "deleted_at" datetime null,
+    "type_id" integer not null,
+    "group_id" integer null,
```

Comment:
Are all the columns above relevant to the test? Please reduce the declaration to the necessary minimum.